fix bug: getAll missing observations

annooritdept opened the pull request:
The shortcut on lines 67-69 LabOrderResultsServiceImpl.java causes an unexcpected result where, when getAll has numberOfAccessions set, observations for orders will only be attached to orders if their encounter is before the last encounter reached by this loop. When using numberOfAccessions, this causes orders to be returned without their associated observed values. The solution seems to be to just remove the shortcut. I haven't tested this.

annooritdept committed:
fix bug getAll missing observations

The shortcut on lines 67-69 cause an unexcpected result where, when getAll has numberOfAccessions set, observations for orders will only be attached to orders if their encounter is dated after the last encounter reached by this loop. The solution seems to be to just remove the shortcut.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Bahmni/bahmni-core?pullRequest=138) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/Bahmni/bahmni-core?pullRequest=138) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Bahmni/bahmni-core?pullRequest=138) it.</sub>

annooritdept committed:
Update LabOrderResultsServiceImpl.java

Previous bugfix was wrong, would have ignored numberOfAccessions altogether. This is possibly better. I apologize, I don't have a development system set up to test this.
